### PR TITLE
#62 62 finish font interface

### PR DESCRIPTION
**Pull request by baozelin**
font interface and font format.

### CODE REVIEW

File: src/main/java/ui/FontFrame.java [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+	
+	JComboBox familyChooseBox;
+	JComboBox styleChooseBox;
+	JComboBox sizeChooseBox;
+
+	/**
+	 * Launch the application.
+	 */
+	/*
+	public static void main(String[] args) {
```
Comment (Weizitong) [inline src/main/java/ui/FontFrame.java]: As I mentioned before, you can delete those useless code

File: src/main/java/ui/FontFrame.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+		contentPane.setLayout(null);
+		
+		JButton yesButton = new JButton("Yes");
+		yesButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				String family = familyChooseBox.getSelectedItem().toString();
+				int size = Integer.parseInt(sizeChooseBox.getSelectedItem().toString());
+				String style = styleChooseBox.getSelectedItem().toString();
+				
+				 switch (style) {
```
Comment (Weizitong) [inline src/main/java/ui/FontFrame.java]: 1. Code style here is wired. Can you please adjust it.
2. Add default to handle unexpected exception.

File: src/main/java/ui/TextEditorUI.java [hunk 3]
```diff
@@ -202,6 +205,20 @@ private void initUI() {
         helpMenu.add(openCooperationAction);
         windowMenu.add(minimizeAction);
         windowMenu.add(zoomAction);
+
+
+
+        fontAction = new JMenuItem("Font");
+        fontAction.addActionListener(new ActionListener() {
```
Comment (Weizitong) [inline src/main/java/ui/TextEditorUI.java]: Please use lamda function here, e.g. addActionListener(e -> {
    // your code here
});

Comment (baozelin) [reply src/main/java/ui/TextEditorUI.java]: ok.

**Pull request merged**